code/workspaces/auth-service/src/dataaccess/userRolesRepository.js

```diff
@@ -28,16 +28,28 @@ function addDefaults(roles) {
 async function getRoles(userId, userName) {
   let roles = await UserRoles().findOne({ userId }).exec();
   if (!roles) {
-    roles = await addRecordForNewUser(userId, userName);
+    roles = await createNonVerifiedUserRecord(userId, userName);
   }
 
   return addDefaults(roles);
 }
 
+// User has logged in for the first time, set up userRoles database record if does not exist
+// and set the user as verified
+async function createNonVerifiedUserRecord(userId, userName) {
+  return addRecordForNewUser(userId, userName, true);
+}
+
+// User has not logged in, however their e-mail is being added to an existing project, hence
+// set up userRoles record and set as not verified
+async function createVerifiedUserRecord(userId, userName) {
```

Comment:
@iwalmsley see other comment about switching the names of createNonVerifiedUserRecord() and createVerifiedUserRecord()